Source/Microsoft.Teams.Apps.FAQPlusPlus/Bots/FaqPlusPlusBot.cs

```diff
@@ -787,9 +788,18 @@ private static Task<TaskModuleResponse> GetTaskModuleResponseAsync(Attachment qu
                     break;
 
                 case Constants.ShareFeedback:
-                    this.logger.LogInformation("Sending user feedback card");
-                    await turnContext.SendActivityAsync(MessageFactory.Attachment(ShareFeedbackCard.GetCard())).ConfigureAwait(false);
-                    break;
+                    if (this.configurationProvider.GetSavedEntityDetailAsync(ConfigurationEntityTypes.FeedbackTeamId).Equals(string.Empty))
+                    {
+                        this.logger.LogInformation("Feedback Team ID not registered yet.");
+                        await turnContext.SendActivityAsync(MessageFactory.Text("I'm sorry, feedback team has not been registered yet."));
```

Comment:
The string "I'm sorry, feedback team has not been registered yet" should be like this:

_I'm sorry, I don't think I can find the Team ID for submitting Feedback_

Also, make sure that the string is saved as part of the RESX file.